internal/routes/all_routes.go

```diff
@@ -79,8 +79,11 @@ func MountAPI(r *chi.Mux) {
 			// Different types do have different payloads, therefore TYPE must be part of
 			// URL and not a URL (filter) parameter.
 			r.Route("/{TYPE}", func(r chi.Router) {
-				r.Get("/{ID}", s.GetReservationDetail)
 				r.Post("/", s.CreateReservation)
+				r.Route("/{ID}", func(r chi.Router) {
+					r.Get("/", s.GetReservationDetail)
+					r.Get("/instances", s.ListInstancesDescription)
+				})
```

Comment:
This will need an OpenAPI spec refresh.

I vote for adding some "This endpoint is temporary" documentation so we are able to remove it in the future, I feel like making our service to be inventory is not the direction we should be going forward long-term.

Reply:
Can you please explain why is it a temporary endpoint?

Reply:
Sure, we believe that our service should not serve as an inventory. This information (public IP, DNS) should come from subscription manager directly into the inventory, it already sends AWS account and instance ID. I will send you a document in PM.